assets/js/base/context/providers/cart-checkout/checkout-processor.ts

```diff
@@ -293,12 +295,20 @@ const CheckoutProcessor = () => {
 							__internalProcessCheckoutResponse( response );
 						} );
 				} catch {
+					let errorMessage = __(
+						'Something went wrong, but your order may still have been placed. Please check your email inbox before retrying.',
+						'woo-gutenberg-products-block'
+					);
+
+					if ( customerId !== 0 ) {
+						errorMessage = __(
+							'Something went wrong, but your order may still have been placed. Please check your email inbox or your order history in the "My Account" area before retrying.',
```

Comment:
Something went wrong when placing the order. Check your account's order history for order updates before retrying.

Reply:
I think we should still include email here, but otherwise fine. Let me update them.